src/components/VisualElement/VisualElementWrapper.tsx

```diff
@@ -24,6 +25,14 @@ interface Props {
   visualElement: GQLVisualElementWrapper_VisualElementFragment;
 }
 
+const StyledFigure = styled(Figure)`
+  margin: 0;
+  width: 100% !important;
+  height: 100%;
+  right: auto !important;
+  left: auto !important;
```

Comment:
Veldig masse !important her

Reply:
Det er fordi det er viktig kode!

Reply:
Det morsomme er at når det står !important så tenker eg "not important" men det er jo det motsatte!